pkg/cephfs/driver.go

```diff
@@ -105,6 +105,10 @@ func (fs *Driver) Run(driverName, nodeID, endpoint, volumeMounter string, cacheP
 		klog.Fatalf("failed to write ceph configuration file: %v", err)
 	}
 
+	if err := remountHisMountedPath(driverName, version, nodeID, cachePersister); err != nil {
```

Comment:
How about wrapping this if-statement in `if MountCacheDir != "" { ... }`?

Reply:
Ah I see, it's already done in that method. I'm not too keen on using exported global variables though.

Reply:
https://github.com/huaizong/ceph-csi/blob/improve-remount-pv-path-when-exit-v2/pkg/cephfs/driver.go#L108-L114

```
	initVolumeMountCache(driverName, mountCacheDir, cachePersister)
	if mountCacheDir != "" {
		if err := remountCachedVolumes(); err != nil {
			klog.Warningf("failed to remount cached volumes: %v", err)
			//ignore remount fail
		}
	}
```